crates/agglayer-config/src/lib.rs

```diff
@@ -115,6 +116,9 @@ pub struct Config {
     #[serde(default)]
     #[serde(skip_serializing_if = "is_false")]
     pub mock_verifier: bool,
+
+    #[serde(default, skip_serializing_if = "crate::default")]
```

Comment:
nit: I think this reads better, I know you didn't write this function
```suggestion
    #[serde(default, skip_serializing_if = "crate::is_default")]
```